rust/private/utils.bzl

```diff
@@ -379,3 +388,80 @@ def transform_deps(deps, make_rust_providers_target_independent):
         build_info = dep[BuildInfo] if BuildInfo in dep else None,
         cc_info = dep[CcInfo] if CcInfo in dep else None,
     ) for dep in deps] if make_rust_providers_target_independent else deps
+
+def is_third_party_package(package, third_party_dir):
+    """Determines if the given package is considered third-party.
```

Comment:
Please add brief comment on what does it mean "third-party" -> "names of third-party crates do not contain the full label" or something in that spirit.

Reply:
Done.